config-local.php-dist

```diff
@@ -6,11 +6,13 @@
  */
 
 // LDAP
-define('LDAP_HOST', 'pm-ns.mozilla.org');
+define('LDAP_HOST', 'DEFAULT_LDAP_HOST');
```

Comment:
The old pm-ns hostname is gradually being removed from all servers, and we do not plan to provide a replacement. Services that use LDAP will need to use the hostname provided to them by Mozilla IT, and there is no default answer to that question that will remain guaranteed to work globally (or internally).